docs/products/kafka/howto/viewing-resetting-offset.rst

```diff
@@ -0,0 +1,145 @@
+View and reset consumer group offsets
+============================================
+
+The `open source Apache Kafka code <https://kafka.apache.org/downloads>`_ includes a ``kafka-consumer-groups.sh`` utility enabling you to view and manipulate the state of consumer groups. 
+Before using the tool, few setup steps need to be performed.
+
+Define the configuration file
+-----------------------------
+
+#. Create the Java keystore and truststore for your Aiven for Apache Kafka service using the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`.
+
+#. Create a ``consumer.properties`` properties file pointing to the keystore and truststore files with the following entries:
+
+* ``security.protocol``: security protocol, SSL for the default TLS security settings
+* ``ssl.keystore.type``: keystore type, ``PKCS12`` for the keystore generated with the :ref:`dedicated Aiven CLI command <avn_service_user_kafka_java_creds>`
+* ``ssl.keystore.location``: keystore location on the file system
+* ``ssl.keystore.password``: keystore password
+* ``ssl.truststore.location``: truststore location on the file system
+* ``ssl.truststore.password``: truststore password
+* ``ssl.key.password``: keystore password
+
+.. Tip::
+
+    The ``avn service user-kafka-java-creds`` :ref:`Aiven CLI command <avn_service_user_kafka_java_creds>` accepts a ``--password`` parameter setting the same password for the truststore, keystore and key
+   
+An example of the ``consumer.properties`` content is the following::
+
+    security.protocol=SSL
+    ssl.keystore.type=PKCS12
+    ssl.keystore.location=client.keystore.p12
+    ssl.keystore.password=changeit
+    ssl.key.password=changeit
+    ssl.truststore.location=client.truststore.jks
+    ssl.truststore.password=changeit
+
+
+Managing consumer group offsets with ``kafka-consumer-groups.sh``
+-----------------------------------------------------------------
+
+The ``kafka-consumer-groups.sh`` tool enables to manage consumer group offsets, the following commands are available.
+
+List the active consumer groups
+'''''''''''''''''''''''''''''''''''''''''
+
+To list the currently active consumer groups use the following command replacing the ``demo-kafka.my-project.aivencloud.com:17072`` with the Aiven for Apache Kafka service URI:
```

Comment:
can we use instead of  "with the Aiven for Apache Kafka " -  "with your service URI", it is easier to read and context is clear. and since we don't mention work Kafka we have no legal issues here.